source/MaterialXContrib/MaterialXMaya/README.md

```diff
@@ -92,21 +100,49 @@ For example:
 reloadMaterialXNode Jade_SR_jade
 ```
 
-Reloads the document and refreshes the viewport shader mapped to the specified MaterialX node. This command is useful when the contents of the document file have changed on disk, e.g. due to editing in an external XML editor.
+Reloads the document and refreshes the viewport shader mapped to the specified MaterialX node.
+This command is useful when the contents of the document file have changed on disk, e.g. due to editing in an external XML editor.
+
+### `reloadMaterialXLibraries`
+
+This command has no arguments.
+When executed, it reloads all MaterialX libraries used by the plug-in (see below for details).
 
 ## MaterialX Libraries
 
-The plug-in loads a set of standard MaterialX libraries (installed with the plug-in) at load time. When a document is loaded for a MaterialX node, it imports these libraries from memory instead of reading them from the file system.
+MaterialX documents can reference elements defined in other documents called libraries.
+When the `MaterialXMaya` plug-in is loaded by Maya, it loads a set of standard libraries
+(installed with the plug-in) and optionally a set of user libraries which can be configured via Maya option variables.
+
+Option variable `materialXLibrarySearchPaths` stores an array of file paths that are used to search for user libraries.
+
+Option variable `materialXLibraryNames` stores an array of directory names that the plug-in searches for under the paths defined by `materialXLibrarySearchPaths`.
+The plug-in loads all `.mtlx` document files found in these directories as MaterialX libraries.
 
-Users can configure additional library search paths and library directory names with the option variables `materialXLibrarySearchPaths` and `materialXLibraryNames` respectively which both hold arrays of strings. 
-The following are a couple of examples in MEL:
+Here's an example of how the two option variables can be configured in MEL:
 
 ```MEL
-optionVar -stringValueAppend materialXLibrarySearchPaths "D:/MaterialX/Teapot_Demo_MatX"
+optionVar -stringValueAppend materialXLibrarySearchPaths "D:/MaterialX/Demo/Libraries"
 optionVar -stringValueAppend materialXLibraryNames "stdlib"
 ```
 
-## Editor integration
+Please note that library and per-node document loading are independent of each other. When a MaterialX document is loaded for a `MaterialXSurfaceNode` or `MaterialXTextureNode`, it imports elements from the libraries currently in memory which speeds up document loading.
+
+On the other hand, all libraries can be reloaded explicitly with the `reloadMaterialXLibraries` Maya command which does not affect the currently loaded documents. The updated libraries will be imported by documents loaded or reloaded afterwards.
+
+## Resource Search Paths
```

Comment:
Another new feature: resource search paths.

Reply:
How about "The updated libraries for these documents will be imported when the documents themselves are loaded or reloaded."

Reply:
I think "afterwards" is the important part here (I would welcome a better expression though): once the libraries have been updated, they are going to be used by all subsequent document "loads". Also, the documents in this sentence are not necessarily the same as in the "currently loaded documents" in the previous sentence.

Reply:
Why don't we just use the words that you suggested "subsequent document loads": 

The updated libraries will be imported upon subsequent document loading and reloading.

Reply:
What do you think about the fix I just committed? Just changing "afterwards" to "subsequently". I think "imported by documents" is more specific because libraries are imported to or by documents.

Reply:
How about: "The updated libraries will be imported by documents that are subsequently loaded or reloaded." ?